index.html

```diff
@@ -14,8 +14,8 @@
     <section id='head'>
```

Comment:
Having an ID of "head" could be problematic and confusing in your code. Also, instead of having a section here, it's best to use the <header> for your page as a direct child of the <body>.